docs/snippets/all/descriptors/descr_builtin_archetype.rs

```diff
@@ -0,0 +1,65 @@
+use rerun::{ChunkStore, ChunkStoreConfig, ComponentDescriptor, VersionPolicy};
+
+#[allow(clippy::unwrap_used)]
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    const APP_ID: &str = "rerun_example_descriptors_builtin_archetype";
+
+    let rec = rerun::RecordingStreamBuilder::new(APP_ID).spawn()?;
+
+    rec.log_static(
+        "data",
+        &rerun::Points3D::new([(1.0, 2.0, 3.0)]).with_radii([0.3, 0.2, 0.1]),
+    )?;
+
+    // When this snippet runs through the snippet comparison machinery, this environment variable
```

Comment:
that's a really weird hack to do the checking. Nothing about this is `/docs/` anymore :/
wouldn't the older "roundtrip tests" do the same job just as well?

if not can we maybe at least add a short readme to this folder to point out that these are not really snippets for documenting but rather tests?

Reply:
I'll see if i can move it to roundtrips/

Reply:
oh no wait, i remember why I picked snippets/ -- these _are_ docs, they show how to override tags in different scenarios! but yeah the presence of the testing code in the rust version kinda suck, hmm...